src/routes/users/handlers.js

```diff
@@ -68,3 +69,23 @@ export async function verifyEmailToken(req, res) {
     await handleErrors(req, res, error, { namespace: 'SERVICE:USER' });
   }
 }
+
+/**
+ * Handler for the active users csv download.
+ *
+ * @param {*} req - request
+ * @param {*} res - response
+ * @returns {*} - active users in a CSV format
+ */
+export async function getActiveUsers(req, res) {
+  try {
+    const usersStream = await activeUsers();
+
+    res.writeHead(200, { 'Content-Type': 'text/csv; charset=utf-8' });
+
+    usersStream.on('end', () => res.end());
+    usersStream.pipe(res);
```

Comment:
I think there should be a check that the user is an admin

Reply:
Added.